feat(interlink): private beta doc (#3942)
scaleway/docs-content: network/interlink/reference-content/statuses.mdx

---
meta:
  title: Understanding InterLink statuses
  description: Find out what the different possible statuses of your InterLink mean, and how to take action based on these statuses when necessary.
  noindex: true
content:
  h1: Understanding InterLink statuses
  paragraph: Find out what the different possible statuses of your InterLink mean, and how to take action based on these statuses when necessary.
tags: interlink status
categories: 
  - interlink
  - network
dates:
  validation: 2024-10-24
  posted: 2024-10-24
---

<Message type="important">
InterLink is currently in [Private Beta](https://www.scaleway.com/en/betas/), and only available to selected testers.
</Message>

> An InterLink always has a **status**, which can be retrieved via the API using the [Get a link](https://scaleway-developers-production-internal.s3-website.fr-par.internal.scaleway.com/en/developers/api/interlink/#path-links-get-a-link) call.

Reviewer: Remove internal preview link
```suggestion
An InterLink always has a **status**, which can be retrieved via the API using the [Get a link](https://www.scaleway.com/en/developers/api/interlink/#path-links-get-a-link) call.
```

This document explains the different statuses possible for an InterLink, and how to understand them.

| **Status**            | **Description**                     | 
|-------------------|-----------------------------------------|
| **Configuring**       | InterLink is configuring and is in a transient state. No user actions can be carried out. <br/><br/> This status could occur immediately after creating your InterLink order, while Scaleway processes the order and configures the InterLink, or at any time when a new configuration is being applied. |

> | **Requesting**        | InterLink has been created successfully on the Scaleway side, provisioning must now be requested from the chosen partner. <br/><br/> You must contact your partner to request that they provision your InterLink. Either a contact email address, or a link to the partner’s portal is provided via the Scaleway API. Scaleway does not make the request to the partner, this is the responsibility of the user.<br/><br/> The InterLink will remain in `Requesting` status until the partner accepts the request, at which point it moves into `Provisioning` status. Alternatively, the partner may refuse the request, or the the request may time out. |

Reviewer 2: ```suggestion
| **Requesting**        | InterLink has been created successfully on the Scaleway side, provisioning must now be requested from the chosen partner. <br/><br/> You must contact your partner to request that they provision your InterLink. Either a contact email address, or a link to the partner’s portal is provided via the Scaleway API. Scaleway does not make the request to the partner, this is the responsibility of the user.<br/><br/> The InterLink will remain in `Requesting` status until the partner accepts the request, at which point it moves into `Provisioning` status. Alternatively, the partner may refuse the request, or the request may time out. |
```

| **Expired**           | No response was received from the selected partner in the 10 days following the creation of the InterLink on the Scaleway side. <br/><br/> If Scaleway does not receive a response from the selected partner after 10 days, the InterLink moves to `Expired` status. It is not possible to take action to move this InterLink to a different status: you must delete the InterLink, and create a new order if you wish to try again. |
| **Refused**           | The partner refused the request to provision the InterLink. <br/><br/> It is not possible to take action to move this InterLink to a different status: you must delete the InterLink, and create a new order if you wish to try again with a different partner or configuration.  |
| **Active**            | The InterLink is in a ready state, and traffic can flow. <br/><br/> The InterLink has been provisioned on both the Scaleway side and the partner side, and is now in an active state. Traffic can flow across the InterLink (presuming that allowed routes lists, route propagation, and an attached VPC are correctly configured). |

> | **Limited Connectivity** | The InterLink is provisioned, but IP connectivity is limited. <br/><br/> This may be the case if either, but not both of, the BGP IPv4 or IPv6 sessions are down. |

Reviewer 2: ```suggestion
| **Limited Connectivity** | The InterLink is provisioned, but IP connectivity is limited. <br/><br/> This may be the case if either, but not both of, the BGP, IPv4, or IPv6 sessions are down. |
```

| **Down / All down**  | The InterLink is provisioned, but there is no IP connectivity. <br/><br/> The InterLink has been provisioned successfully on both sides, but both BGP sessions (IPv4 and IPv6) are down and traffic cannot flow. Open a support ticket with Scaleway, or contact your partner. |

> | **Locked** | The InterLink has been locked by the Trust and Safety team <br/><br/> You cannot carry out any actions on the InterLink. Open a support ticket.|

Reviewer: ```suggestion
| **Locked** | The InterLink has been locked by the Trust and Safety team <br/><br/> You cannot carry out any actions on the InterLink. Open a support ticket. |
```
